### DISCUSSION TITLE
Minor chord in Chromatic scale ?

### DISCUSSION THREAD

**Discussion by SlikkTim** [bangcorrupt/hacktribe#166] [Q&A] [answered]
Hey there, I have no idea where to ask for this, but could there be a way to hack the E2 so the chromatic scale in Chord mode would make minor chords instead of major chords ?

It's kind of a key element in techno.

Unrelated, if there would be a way to modify just that one feature as I need all the synthesis on the E2... That would be amazing.

All the best
T

**Comment by bangcorrupt** [comment, marked as answer]
Hi.  When I work on scale mode again I'll make chord sets configurable.  Hopefully someday all the features from synth and sampler will be available in hacktribe.
